README.md

```diff
@@ -1 +1,65 @@
 # scpca-paper-figures
+
+This repo contains the figures and tables included in the ScPCA manuscript.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Summary of figures](#summary-of-figures)
+- [Generating figures](#generating-figures)
+- [Sample info](#sample-info)
+- [Color palettes](#color-palettes)
+- [Renv](#renv)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Summary of figures
+
+Below is a summary of all figures in the paper.
+
+**Figure 1**
+
+A. Bar chart summarizing the types of diagnoses found on the Portal.
+B. Bar chart summarizing the types of modalities found on the Portal.
```

Comment:
Sadly this isn't coming out as bullet list. Might need to do:

```
- Panel A: ...
- Panel B: ...
```

Reply:
I wasn't actually trying to make a bulleted list, but I can.

Reply:
I dont think it needs to be bullet, just is hard to see now: `A. Bar chart summarizing the types of diagnoses found on the Portal. B. Bar chart summarizing the types of modalities found on the Portal.`

Maybe a semicolon!

Reply:
Oh you are right, I forgot it wouldn't just magically be on a new line.